exercises/spiral/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "fmt"
+
+func Spiral(x, y int) [][]int {
```

Comment:
This is wrong implementation. The tests are failing. You can check this by running `go test -v ./exercises/spiral`.